src/main/java/com/sintern/api/request/ApplicationByOpenInternPositionAndStudentRequest.java

```diff
@@ -0,0 +1,25 @@
+package com.sintern.api.request;
+
+import java.util.UUID;
+
+public class ApplicationByOpenInternPositionAndStudentRequest {
+
+    private UUID openInternPositionID;
+    private UUID studentID;
+
+    public UUID getStudentID() {
+        return studentID;
+    }
+
+    public void setStudentID(UUID studentID) {
+        this.studentID = studentID;
+    }
+
+    public UUID getOpenInternPositionID() {
+        return openInternPositionID;
+    }
+
+    public void setOpenInternPositionID(UUID openInternPositionID) {
+        this.openInternPositionID = openInternPositionID;
+    }
```

Comment:
Why don't we use Lombok annotation @ Getter and @ Setter, for cleaner code?

Reply:
Ok, I'm making the change. But all response and request classes from API are like that. So they should be changed? @DascaluCosmin

Reply:
It's fine to keep them as they are.